src/Pages/Main/Main.js

```diff
@@ -1,7 +1,56 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import Cards from './Cards';
 
 function Main() {
-  return <div></div>;
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
+    getCards();
+  }, []);
+
+  const getCards = () => {
+    fetch('/data/cardlist.json')
+      .then(res => res.json())
+      .then(data => setCards(data.cards));
+  };
+
+  return (
+    <div>
+      <Container>
+        <Row>
+          {cards.map((cards, id) => (
+            <Cards
+              key={id}
+              title={cards.title}
+              price={Math.floor(cards.price)
+                .toString()
+                .replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
+              bgImg={cards.bgImg}
+              userImg={cards.userImg}
+              name={cards.name}
+              nick={cards.nick}
+              liketext={cards.liketext}
```

Comment:
```suggestion
              price={Math.floor(cards.price)
                .toString()
                .replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
             card={cards}
```
cards의 모든 데이터를 다 넘길거면 굳이 하나씩 분리해서 넘길필요없이 한번에 객체 전체를 넘겨도 되지 않을까요?